kola/harness.go

```diff
@@ -206,6 +206,22 @@ func filterTests(tests map[string]*register.Test, pattern, pltfrm string, versio
 				blacklisted = true
 				break
 			}
+
+			// Check if any native tests are blacklisted. To exclude native tests, specify the basic
+			// test and a "/" and then the glob pattern. Example: rhcos.basic/TestNetworkScripts
```

Comment:
What's the intended behavior if there is no glob pattern after the `/`? I believe right now this logic would panic.

Reply:
in case of something like `basic/`, it would run all the tests because `bl[nativeblacklistindex+1:]` would return an empty string. I will call that out explicitly in the docstring.